client/src/components/ProductDetails/index.jsx

```diff
@@ -21,11 +21,13 @@ const ProductDetails = ({ productId }) => {
     }
   }, [productId]);
```

Comment:
Hey since product ID is being passed in as a prop, do you need to track its changes with a useEffect? when it is changed outside of our component wont the prop get re passed down with react?